Remotion/Web/Development.WebTesting/WebTestHelper.cs

```diff
@@ -190,12 +191,13 @@ public void OnSetUp ([NotNull] string testName)
     /// Creates a new browser session using the configured settings from <see cref="WebTestConfigurationFactory"/>.
     /// </summary>
     /// <param name="maximizeWindow">Specified whether the new browser session's window should be maximized.</param>
+    /// <param name="options">Specifies additional options applied when creating the browser.</param>
     /// <returns>The new browser session.</returns>
-    public IBrowserSession CreateNewBrowserSession (bool maximizeWindow = true)
+    public IBrowserSession CreateNewBrowserSession (bool maximizeWindow = true, [CanBeNull] DriverOptions options = null)

```

Comment:
Do we Use CanBeNull on optional Parameter? Not sure about that, as the Nullability is somewhat give. But no idea if ReSharper knows that. But if yes, its missing in OnFixtureSetup

Reply:
Found many optional arguments having a `[CanBeNull]` attribute. Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/6/commits/e3624ba3785899bc6fc3f9b84eb47d2c6b892c1a.